### PR TITLE
Explore Metrics: Duplicate subpath

### LINKED ISSUE #96269: Explore Metrics: Duplicate subpath for generated explore link
### What happened?

We serve our Grafana on the `/grafana` subpath, and everything works just fine except for this reported issue within Explore Metrics.
When I do the steps described in the "How do we reproduce it?" section, then the generated URI is invalid because it contains a double subpath, like `company.org/grafana/grafana/explore?` instead of just `company.org/grafana/explore?`.

### What did you expect to happen?

Generated URI with proper, single subpath.

### Did this work before?

Not sure.

### How do we reproduce it?

1. Use the Explore metrics feature
2. Filter any metrics
3. Click on "open in explore" (see figure) and check the URI
Image

### Is the bug inside a dashboard panel?

No

### Environment (with versions)?

Grafana: v11.3.0 (d9455ff7db)
OS: Linux/Ubuntu 22.04
Browser: Chrome

### Grafana platform?

Kubernetes

### Datasource(s)?

_No response_

### PR DESCRIPTION
**Pull request by yincongcyincong**
Fixes #96269

### CODE REVIEW

**Comment by yincongcyincong** [comment]
@adrapereira bro. could you please take a look this pr. thanks.

**Comment by itsmylife** [comment]
@bohandley I found this PR https://github.com/grafana/grafana/pull/94525
I think this handles the subPath handling already. Because `getLinkToExplore()` function is already calling [`getExploreUrl`](https://github.com/grafana/grafana/blob/315778227b010f2d929ebbf9d8617154f4f1373c/public/app/core/utils/explore.ts#L64-L99) function from `explore.ts`.

File: public/app/features/trails/MetricScene.tsx [hunk 1]
```diff
@@ -165,7 +165,7 @@ export class MetricActionBar extends SceneObjectBase<MetricActionBarState> {
     reportExploreMetrics('selected_metric_action_clicked', { action: 'open_in_explore' });
     this.getLinkToExplore().then((link) => {
       // We need to ensure we prefix with the appSubUrl for environments that don't host grafana at the root.
-      const url = `${config.appSubUrl}${link}`;
+      const url = `${link}`;
```
Comment (itsmylife) [inline public/app/features/trails/MetricScene.tsx:168]: Now we are using link directly so we can just pass it to the `window.open`. No need to define a parameter.

**Review by itsmylife — CHANGES_REQUESTED** [review]
A small nit-pick then I think it's good to go.

File: public/app/features/trails/MetricScene.tsx (hunk 1 above)
Comment (bohandley) [reply public/app/features/trails/MetricScene.tsx:168]: Agreed! LGTM
`window.open(link, '_blank');`

**Review by bohandley — APPROVED** [review]
Testing with this in `custom.ini` for the configured sub_path. The extra  part of the full facing root_url does work locally. LGTM! 🚀 

```
# The full public facing url
root_url = %(protocol)s://%(domain)s:%(http_port)s/extra/

# Serve Grafana from subpath specified in `root_url` setting. By default it is set to `false` for compatibility reasons.
serve_from_sub_path = true
```

Screenshot 2024-11-15 at 8 16 41 AM

**Comment by yincongcyincong** [comment]
@itsmylife @bohandley thanks for youre review. i modified the code. could you please take a look.

**Review by itsmylife — APPROVED** [review]

**Pull request merged**